.github/workflows/eest-docs.yaml

```
@@ -0,0 +1,50 @@
+name: Spec Docs
+
+on:
+  push:
+    branches:
+      - mainnet
+      - 'forks/**'
+    paths:
+      - 'src/ethereum_spec_tests/**'
+      - 'tests/eest/**'
+      - '.github/workflows/eest-docs.yaml'
```

```suggestion
      - '.github/workflows/testing-docs.yaml'
```
If the comment to the file name is applied.